### PR TITLE
changed first sentence for parallel structure

### PR DESCRIPTION
**Pull request by TalyaF**
/assign @chenopis

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4304)

### CODE REVIEW

**Comment by chenopis** [comment]
@TalyaF Let me know if you have trouble w/ the CNCF CLA process. This is one of the fictions points of the onboarding process, so if you could document any trouble you have w/ this and brief me, that would be very helpful. Thx!

**Comment by TalyaF** [comment]
No, it was pretty straightforward. And the document has been signed. 

Thanks so much for you help today!

File: docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This page is an overview of Kubernetes.
 {% endcapture %}
 
 {% capture body %}
-Kubernetes is an [open-source platform for automating deployment, scaling, and operations of application containers](http://www.slideshare.net/BrianGrant11/wso2con-us-2015-kubernetes-a-platform-for-automating-deployment-scaling-and-operations) across clusters of hosts, providing container-centric infrastructure.
+Kubernetes is an [open-source platform designed to automate deploying, scaling, and operating application containers.]  (http://www.slideshare.net/BrianGrant11/wso2con-us-2015-kubernetes-a-platform-for-automating-deployment-scaling-and-operations)
```
Comment (chenopis) [inline docs/concepts/overview/what-is-kubernetes.md]: You'll need to remove the spaces between `]` and `(`.

**Comment by chenopis** [comment]
@TalyaF Just one small issue you'll need to correct. There's an extra space on line 15 that is breaking the link.

**Pull request merged**